inference-engine/src/inference_engine/compilation_context.hpp

```diff
@@ -30,10 +30,12 @@ struct NetworkCompilationContext final {
         const std::map<std::string, std::string> & compileOptions = {}) :
             m_weights{},
             m_compileOptions{compileOptions},
+            m_modelName{network.getName()},
```

Comment:
I suppose it's excess in this case, since model name is already in serialized representation after `ngraph::pass::Serialize::run_on_function`